docs/developer-guides/using-docker.mdx

```diff
@@ -0,0 +1,79 @@
+---
+id: using-docker
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+# Using Docker
+
+SuperBench runs benchmarks inside Docker container,
+here's the guide on how to build and start containers during development.
+
+## Build image
+
+You need to [clone the code](./development#set-up) first before building the image.
+
+<Tabs
+  groupId='gpu-platform'
+  defaultValue='cuda'
+  values={[
+    {label: 'CUDA', value: 'cuda'},
+    {label: 'ROCm', value: 'rocm'},
+  ]
+}>
+<TabItem value='cuda'>
+
+```bash
+export DOCKER_BUILDKIT=1
+docker buildx build \
+  --platform linux/amd64 --cache-to type=inline,mode=max \
+  --tag superbench-dev --file dockerfile/cuda11.1.1.dockerfile .
+```
```

Comment:
our build pipeline uses the same command?

Reply:
similar, it will add extra `--label` parameters for creation time, revision, etc. don't affect actual building